recipes/cnmaps/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "cnmaps" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Clarmy/cnmaps/archive/refs/tags/0.2.1.tar.gz
+  sha256: 4f35acf5d17b0be4df67745ae2c59201241eeba6df9a75f04df8889f7bf74c90
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib-base
+    - Cartopy >=0.19.0
+    - Shapely
+    - numpy
+    - netCDF4
+
+test:
+  imports:
+    - cnmaps
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/Clarmy/cnmaps
+  summary: 'cnmaps is a python package that make drawing China maps more easily. '
+  description: |
+    cnmaps is a python package that make drawing China maps more easily. 
+  license: MIT
+  license_family: MIT
```

Comment:
Remember to update the license label here when you update to a new version.

Reply:
OK, Thank you.